src/Context/RestContext.php

```diff
@@ -132,10 +132,11 @@ public function theHeaderShouldNotBeEqualTo($name, $value) {
      *
      * @Then the header :name should contain :value
      */
-    public function theHeaderShouldBeContains($name, $value)
+    public function theHeaderShouldContain($name, $value)
     {
-        $this->assertContains($value, $this->request->getHttpHeader($name),
-            "The header '$name' doesn't contain '$value'"
+        $actual = $this->request->getHttpHeader($name);
+        $this->assertContains($value, $actual,
+            "The header '$name' value is '$actual'"
```

Comment:
Same as above, now missing the expected value.
Maybe something like `"The header '$name' doesn't contain '$value', it is: $actual"`